pom.xml

```diff
@@ -86,8 +86,8 @@ under the License.
         <paimon-flinkx-common>paimon-flink1-common</paimon-flinkx-common>
         <paimon-flink-common.flink.version>1.20.1</paimon-flink-common.flink.version>
         <flink.scala.binary.version>2.12</flink.scala.binary.version>
-        <scala212.version>2.12.15</scala212.version>
```

Comment:
I prefer to use `2.12.18` which is same with the latest scala version that Spark 3.5 use.